.env.example

```diff
@@ -8,4 +8,10 @@ TABLE_NAME=test-table
 
 # PROJECT CONFIG
 OBJECTIVE=Solve world hunger
-FIRST_TASK=Develop a task list
\ No newline at end of file
+# For backwards compatibility
+# FIRST_TASK can be used instead of INITIAL_TASK
+INITIAL_TASK=Develop a task list
+
+# APP CONFIG
+# Set to true to enable command line args support
+ENABLE_COMMAND_LINE_ARGS=false
```

Comment:
Not sure I got what you mean, but... The reason I wanted control from .env is because I want the command-line parsing to be treated as an "extension" of the core babyagi. The goal here is to make it so that people literally can copy only babyagi.py and run it locally and ignore anything else besides the tight core loop.